schema_based_on_diagram.sql

```diff
@@ -0,0 +1,46 @@
+create table patients(
+    id int primary key,
+    name varchar(50),
+    date_of_birth date
+);
+
+create table medical_histories(
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    patient_id int,
+    admitted_at timestamp,
+    status varchar(50),
+    foreign key (patient_id) references patients(id)
+);
+
+create table treatments(
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    type varchar(50),
+    name varchar(50)
+);
+
+create table invoices(
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
+    total_amount decimal,
+    generated_at timestamp,
+    paued_at timestamp,
+    medical_history_id int,
+    foreign KEY (medical_history_id) REFERENCES medical_histories(id)
+)
+
```

Comment:
- [ ] All 'join table' for many to many relationships should be created, kindly make sure to create a table between `medical_histories` and `treatments`
e.g
```sql
 CREATE TABLE medical_histories_has_treatments (
    medical_history_id int refrences medical_histories(id),
    treatment_id int refrences treatments(id),
    );
```